### PR TITLE
Add RichTexture by @stevemoser

### PR DESCRIPTION
**Pull request by stevemoser**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -6864,6 +6864,26 @@
...
+        {
+      "title": "RichTexture",
+      "category-ids": [
+        "text"
+      ],
+      "description": "Rich text editor (fork of Textor but for rich text)",
+      "license": "mit",
+      "source": "https://github.com/stevemoser/richtexture",
+      "homepage": "https://twitter.com/SteveMoser/status/999027011937972224",
+      "itunes": "https://itunes.apple.com/app/richtexture/id1376116077?mt=8",
```
Comment (dkhamsing) [inline contents.json]: Can you remove ?mt=8

File: contents.json [hunk 2]
```diff
@@ -6864,6 +6864,26 @@
...
+    },
+        {
+      "title": "RichTexture",
+      "category-ids": [
+        "text"
+      ],
+      "description": "Rich text editor (fork of Textor but for rich text)",
+      "license": "mit",
+      "source": "https://github.com/stevemoser/richtexture",
+      "homepage": "https://twitter.com/SteveMoser/status/999027011937972224",
```
Comment (dkhamsing) [inline contents.json]: Can you remove this line? Not sure a tweet can be a homepage

File: contents.json [hunk 3]
```diff
@@ -6864,6 +6864,26 @@
...
+      "itunes": "https://itunes.apple.com/app/richtexture/id1376116077?mt=8",
+      "stars": 20,
+      "tags": [
+        "swift"
+      ],
+      "date_added": "Tue Jun 28 18:10:00 2018 +0200",
+      "suggested_by": "@fassko",
```
Comment (dkhamsing) [inline contents.json]: This should be updated to show your github username

**Comment by vpeschenkov** [comment]
Hi @stevemoser! Thank you for the contribution. Could you fix the review comments?

**Comment by stevemoser** [comment]
@vpeschenkov Yes, thank you for the feedback. I'll get to it before the end of tomorrow.

**Pull request merged**